### PR TITLE
Vt logo and icons placing

### PR DESCRIPTION
**Pull request by nyolarraklay**

### CODE REVIEW

**Review by Sovak3441 — CHANGES_REQUESTED** [review]
Please resolve merge conflicts in login.js and editStudent.js

**Review by Sovak3441 — CHANGES_REQUESTED** [review]
Please resolve the Merge conflict in index.html, index.js and login.js.
Make sure the code is synced before any codechanges is made.

**Comment by nyolarraklay** [comment]
It's not really my issue to fix conflicts, but because someone was kind enough to put enhancement on Bootstrap. I fix it as well

**Review by Sovak3441 — APPROVED** [review]

File: src/js/api/auth/login.js [hunk 1]
```diff
@@ -77,3 +43,62 @@ function getRedirectUrl(role) {
...
+         window.location.replace('../../../pages/user/index.html');
+
+        } else if (profile.admin) {
+          new Store('Role', 'admin', Boolean(remember !== 'on'));
+          window.location.replace('#'); // TODO: Add admin page url
+        } else {
+          window.location.replace('../../../pages/user/index.html');
+        }
+        break;
+      case 403:
```
Comment (maleneivy) [inline src/js/api/auth/login.js:85]: There's no switch in this method, so I'm getting a syntax error. Can you fix it? 🙂

Skjermbilde 2024-01-11 kl  17 16 29

Comment (nyolarraklay) [reply src/js/api/auth/login.js:85]: Oh i just change the path of index.html. i didnt do anything i think.

**Pull request merged**